docs/design/03-ports-and-port-bindings.md

```diff
@@ -47,8 +49,8 @@ Config on NXOS leaf:
 ```
 
 
-## Baremetal on Fabric
-Baremetal servers are in most cases directly on dedicated Arista switches which will become part of our fabric.
+## Bare metal on Fabric
```

Comment:
Wouldn't the `m` in `metal` also be uppercase in a title?

Reply:
Will do